kaznet/apps/ona/tests/test_api.py

```diff
@@ -248,10 +251,58 @@ def test_get_xform(self, mocked):
         }
 
         url = urljoin(settings.ONA_BASE_URL, 'api/v1/forms/53')
-        mocked.get(url, json=mocked_xform_data)
+        mocked.get(url, json=mocked_xform_data1)
+
+        # data returned from the 'api/v1/forms/{form_id}/form.json' endpoint
+        mocked_xform_data2 = {
+            "name": "test_task_happy10092019",
+            "type": "survey",
+            "title": "happytoday10092019",
+            "id_string": "happytoday10092019",
+            "sms_keyword": "happytoday10092019",
+            "default_language": "default",
+            "version": "happytoday10092019",
+            "children": []}
+
+        url = urljoin(settings.ONA_BASE_URL, 'api/v1/forms/53/form.json')
+        mocked.get(url, json=mocked_xform_data2)
+
         response = get_xform(53)
 
-        self.assertTrue(response, mocked_xform_data)
+        # data we are supposed to get from get_xform(53) call
+        mocked_xform_data3 = {
```

Comment:
Could we change this to `expected_data` or something ? Seems a bit weird, feels like its a completely different form.

Reply:
and done!